<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="2"
           failures="2"
           errors="0"
           skipped="0"
           time="57.578">
  <properties />
            <testcase name="Registration" time="24.397" classname="com.w2automation.TestCases.TC01_Registration">
                <failure type="java.lang.AssertionError"
                  message="null"
              >
<![CDATA[java.lang.AssertionError: null
  org.testng.Assert.fail(Assert.java:110)
  org.testng.Assert.fail(Assert.java:115)
  com.w2automation.pages.RegistrationPage.clickSignIn(RegistrationPage.java:37)
  com.w2automation.TestCases.TC01_Registration.Registration(TC01_Registration.java:21)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1259)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
            <testcase name="Registration" time="33.181" classname="com.w2automation.TestCases.TC01_Registration">
                <failure type="java.lang.AssertionError"
                  message="null"
              >
<![CDATA[java.lang.AssertionError: null
  org.testng.Assert.fail(Assert.java:110)
  org.testng.Assert.fail(Assert.java:115)
  com.w2automation.pages.RegistrationPage.enterfname(RegistrationPage.java:89)
  com.w2automation.TestCases.TC01_Registration.Registration(TC01_Registration.java:26)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1259)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
